nextcode/config.py

```diff
@@ -109,12 +109,12 @@ def _load_config() -> Dict:
 
 def save_config() -> None:
     config = Config()
-    config_file = root_config_folder.joinpath("config.yaml")
-    log.debug(
-        "Saving config with %s profiles to %s", len(config.get("profiles")), config_file
-    )
+    config_filename = root_config_folder.joinpath("config.yaml")
+    no_profiles = len(config.get("profiles"))
```

Comment:
call this `num_profiles`, or just skip remove it since it's just for the debug log